scripts.py

```diff
@@ -94,6 +95,15 @@ def parse_time(cls, time_string):
                     continue
         raise ValueError("Could not parse time: %s" % time_string)
 
+    def __init__(self, _db=None):
```

Comment:
I refactored this from RunCoverageProviderScript, making it generally possible to pass in the test database session to a Script constructor when testing it.